front/app/components/PostShowComponents/Comments/CommentsSection/PublicComments/index.tsx

```diff
@@ -80,6 +74,7 @@ const PublicComments = ({
   } = useComments({
     ideaId: postId,
     sort: sortOrder,
+    pageSize: 5,
```

Comment:
Could we up this to 10 without much impact so the first load has enough to read?